src/main/java/com/iexec/common/chain/eip712/TypeParam.java

```diff
@@ -28,4 +34,12 @@ public class TypeParam {
     private String name;
     private String type;
 
+    /**
+     * Gets the description of a single parameter as a String.
+     * @return The {@code "type name"} string
```

Comment:
How about
```suggestion
     * @return The {@code "<type> <name>"} string
```
to highlight that both parts are dynamic.

Reply:
done in 843c797